front/src/components/Task/TaskTagChip.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <q-chip
+    class="text-white"
+    :clickable="clickable"
+    :ripple="clickable"
+    :style="style"
+    @click="filterTag(name)"
+  >
+    <span
+      style="max-width: 120px"
+      class="ellipsis"
+    >
+      {{ name }}
+    </span>
+  </q-chip>
+</template>
+
+<script lang="ts">
+import ctfnote from 'src/ctfnote';
+import { injectStrict } from 'src/ctfnote/utils';
+import { defineComponent } from 'vue';
+import keys from '../../injectionKeys';
+export default defineComponent({
+  props: {
+    name: { type: String, default: '?' },
+  },
+  setup() {
+    let filterTag;
+    let clickable;
+
+    // Check if keys.filterTag exists
+    try {
+      filterTag = injectStrict(keys.filterTag);
+      clickable = true;
+    } catch (e) {
+      filterTag = () => {return null;};
+      clickable = false;
+    }
```

Comment:
What is the case that it does not exist?

Reply:
This only exists in the main task list. This code (a large part of which already existed prior to this pull request) adds clicked tags to 'Filter by tag', and that filter does not exist in the task switcher dropdown. Without this statement Vue can't find `keys.filterTag` and would refuse to render the dropdown menu.